cfn-resources/project/test/cfn-test-create-inputs.sh

```diff
@@ -63,13 +65,12 @@ jq --arg org "$MONGODB_ATLAS_ORG_ID" \
 
 jq --arg org "${org_id}" \
 	--arg name "${name}-tags" \
-	'.OrgId?|=$org |.Name?|=$name |.Profile?|=$profile' \
+	'.OrgId?|=$org |.Name?|=$name' \
```

Comment:
For reference I remember using a snippet like the following [search-deployment/test/cfn-test-create-inputs.sh#L22](https://github.com/mongodb/mongodbatlas-cloudformation-resources/blob/master/cfn-resources/search-deployment/test/cfn-test-create-inputs.sh#L22), given that contract test does define a custom profile [workflows/contract-testing.yaml#L191](https://github.com/mongodb/mongodbatlas-cloudformation-resources/blob/master/.github/workflows/contract-testing.yaml#L191).